src/codegen/codegen.test.ts

```diff
@@ -129,7 +130,6 @@ describe('Code generation', () => {
       allowlist: [],
       includeStaticAssets: false,
       scriptName: 'my-script.js',
-      thresholds: [],
```

Comment:
Sneaky fix for a failing typecheck that prevented me from making a commit. I wonder why it the CI checks don't fail :thinking: